src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -78,7 +82,7 @@ __targetDir=$1
 __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
 __toolsDir=$__scriptpath/../Tools
 __dotnet=$__toolsDir/dotnetcli/dotnet
-__packagesDir=$__scriptpath/../packages
+__packagesDir="${NUGET_PACKAGES:-${__scriptpath}/../packages}"
```

Comment:
Did we ever end up putting crossgen into the shared framework? That would make this script a lot simpler.